x/auth/vesting/alias.go

```diff
@@ -0,0 +1,28 @@
+package vesting
```

Comment:
ideally the alias should be autogenerated using the `multitool` (check other `alias.go` files for reference)